packages/client-twitter/src/post.ts

```diff
@@ -459,7 +459,7 @@ export class TwitterPostClient {
                     .replace(/^\s*{?\s*"text":\s*"|"\s*}?\s*$/g, "") // Remove JSON-like wrapper
                     .replace(/^['"](.*)['"]$/g, "$1") // Remove quotes
                     .replace(/\\"/g, '"') // Unescape quotes
-                    .replace(/\\n/g, "\n") // Unescape newlines
+                    .replace(/\\n/g, "\n\n") // Unescape newlines, ensures double spaces
                     .trim();
```

Comment:
Updated the newline handling to ensure double line breaks, improving tweet formatting consistency.